src/main/java/bookstore/mapper/BookMapper.java

```diff
@@ -0,0 +1,21 @@
+package bookstore.mapper;
+
+import bookstore.dto.BookDto;
+import bookstore.dto.CreateBookRequestDto;
+import bookstore.model.Book;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValueCheckStrategy;
+
+@Mapper(componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
+        implementationPackage = "<PACKAGE_NAME>.impl")
+public interface BookMapper {
+    BookDto toDto(Book book);
+
+    Book toBook(CreateBookRequestDto requestDto);
+
+    void updateBookFromDto(CreateBookRequestDto requestDto, @MappingTarget Book book);
```

Comment:
```suggestion
    void updateBook(CreateBookRequestDto requestDto, @MappingTarget Book book);
```

What if CreateBookRequestDto includes null values? Your book will be updated with null? Read about NullValuePropertyMappingStrategy.IGNORE)